server/server.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
+const session = require('express-session');
+const bodyParser = require('body-parser');
+const passport = require('passport');
+const redis = require('connect-redis')(session);
+
+const routes = {
+  board : require('./routes/board')
```

Comment:
Although this works, the usual Node convention is to not use require within an object. I think this ends up just being more code in the long run anyway.